GoogleMulticastAppDelegate/Sources/Public/GoogleUtilities/GULMulticastAppDelegate.h

```diff
@@ -0,0 +1,40 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+
+#import <GoogleUtilities/GULApplication.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol GULMulticastAppDelegateProtocol <NSObject>
+
+- (void)addInterceptorWithDelegate:(id<GULApplicationDelegate>)interceptor;
+
+- (void)removeInterceptorWithDelegate:(id<GULApplicationDelegate>)interceptor;
```

Comment:
IIUC, the passed in delegate will act like an "interceptor", correct? 

nit: If so, I feel like the parameter name should be something like "delegate" or "interceptingDelegate"... wdyt?

Reply:
yeah this should actually be interceptor based on the API review. Google catch.